Firestore/Source/Public/FirebaseFirestore/FIRLoadBundleTask.h

```diff
@@ -28,28 +34,57 @@ typedef NS_ENUM(NSInteger, FIRLoadBundleTaskState) {
 
 } NS_SWIFT_NAME(LoadBundleTaskState);
 
+/** Represents a progress update or a final state from loading bundles. */
 NS_SWIFT_NAME(LoadBundleTaskProgress)
 @interface FIRLoadBundleTaskProgress : NSObject
 
+/** How many documents have been loaded. */
 @property(readonly, nonatomic) NSInteger documentsLoaded;
+
+/** The total number of documents in the bundle. 0 if the bundle failed to parse. */
 @property(readonly, nonatomic) NSInteger totalDocuments;
+
+/** How many bytes have been loaded. */
 @property(readonly, nonatomic) NSInteger bytesLoaded;
+
+/** The total number of bytes in the bundle. 0 if the bundle failed to parse. */
 @property(readonly, nonatomic) NSInteger totalBytes;
 
+/** The current state of `FIRLoadBundleTask` (`LoadBundleTask` in Swift). */
 @property(readonly, nonatomic) FIRLoadBundleTaskState state;
 
 @end
 
+/** A handle associated with registered observers, that can be used to remove them. */
 typedef NSInteger FIRLoadBundleHandle NS_SWIFT_NAME(LoadBundleHandle);
 
+/**
+ * Represents the task of loading a Firestore bundle. Observers can be registered with this task to
+ * observe progresses of bundle loading, as well as task completion and error events.
```

Comment:
Hmm, I'd expect `progresses` to be singular -- `observe the progress` (perhaps `observe the bundle loading progress`), but better check (I think both are correct, it's just a question of which flows better).